src/photos/ducks/timeline-clusters/dates.js

```diff
@@ -20,7 +20,8 @@ import {
  */
 const formatDate = (f, date, formatter) => {
   const [year, month, day] = date.substr(0, 10).split('-')
-  const [hours, minutes, seconds] = date.substr(11, 8).split(':')
+  const [hours, minutes, seconds] =
+    date.length > 10 ? date.substr(11, 8).split(':') : [0, 0, 0]
```

Comment:
Are we sure to also have at least 10 char in the date (to have year/month/day)? Or should we do something similar for that? 

When can we have date without `hours / minutes / seconds` ? Is it because we don't have these informations in the exif datas? 

Since we are setting an arbitrary value to `h:m:s`, do we really want to display 

My Section Title ⠂00h ? 

```
 let titleWithHours = section.title + ' ⠂' + formatH(f, startPeriod) + 'h'
      if (!isSameHour(f, endPeriod, startPeriod)) {
        titleWithHours += '-' + formatH(f, endPeriod) + 'h'
      }
```

Reply:
> Are we sure to also have at least 10 char in the date 
Yes, the format used is always Y-M-D, like `2019-01-01`

> When can we have date without hours / minutes / seconds ? 
When there is a "daily" section, i.e. photos not clustered yet. For these photos, we extract the date from `metadata.datetime` and group them per day, like `2019-01-01` 

>Since we are setting an arbitrary value to h:m:s, do we really want to display
My Section Title ⠂00h ?

We only show the hours when there are more than one cluster in the same day. The `h:m:s` forced to 0 will only apply to photos without clusters, that will be grouped by day